### DISCUSSION TITLE
Raw Mongodb support

### DISCUSSION THREAD

**Discussion by weepy** [SoftwareBrothers/admin-bro#1400] [General]
Am I correct in thinking AdminJS doesn't support Mondodb without mongoose. ?

**Comment by krzysztofstudniarek** [comment]
What kind of support for Mongodb you will look for? 

AdminJS is built in the way you need to inject some kind of ORM, so the application can connect to database. MongoDB conection is supported through Mongoose, Prisma, TypeORM and MikroORM. You can use either of them.
